about/package-scope.md

```diff
@@ -18,21 +18,257 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
+
+### Database software bindings
+
+ Bindings and wrappers for database APIs.
+
+ * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
+
+
+## Domain areas
+
+In addition, our scope includes focused domain areas. These areas are based on
+partnerships that we form with communities and also expertise that we hold
+within our organization. As we develop [new community partnerships](/partners/scientific-communities) and grow,
+we will expand this list.
+
+### Geospatial
+
+Packages focused on the retrieval, manipulation, and analysis of spatial data.
+
+* Examples: [PyGmt](https://github.com/pyOpenSci/software-submission/issues/43),
+[Moving Pandas ](https://github.com/pyOpenSci/software-submission/issues/18)
+
+### Pangeo
+
+We have a [partnership with Pangeo](../partners/pangeo). Often times packages submitted as a part of that partnership are also in the geospatial domain.
+
+* Examples: [xclim - under review now](https://github.com/pyOpenSci/software-submission/issues/73)
+
+### Education
+
+Packages to aid with instruction.
+<!--TODO -  Earthpy fit in this category but it also needs to be sunsetted -->
+
+## Package technical scope
+
+To be in technical scope for a pyOpenSci review, your package:
+
+* Should have maintenance workflows documented.
+* Should declare vendor dependencies using standard approaches rather than including code from other packages within your repository.
+* Should not have an exceedingly complex structure. Others should be able to contribute and/or take over maintenance if needed.
+
+```{admonition} pyOpenSci's goal is to support long(er) term maintenance
+pyOpenSci has a goal of supporting long term maintenance of open source
+Python tools. It is thus important for us to know that if you need to step down as a maintainer, the package infrastructure and documentation is
+in place to support us finding a new maintainer who can take over you
+package's maintenance.
 ```
 
+### Telemetry & user-informed consent
+
+Your package should avoid collecting usage analytics. With
+that in mind, we understand that package-use data can be invaluable for the
+development process. If the package does collect such data, it should do so
+by prioritizing user-informed-consent. This means that before any data are
+collected, the user understands:
+
+1. What data are collected
+2. How the data are collected.
+3. What you plan to do with the data
+4. How and where the data are stored
+
+Once the user is informed of what will be collected and how that data will be handled, stored and used, you can implement `opt-in` consent. `opt-in` means that the user agrees to usage-data collection prior to it being collected (rather than having to opt-out when using your package).
+
+We will evaluate usage data collected by packages on a case-by-case basis
+and reserve the right not to review a package if the data collection is overly
+invasive.
+
+### Notes on scope categories
```

Comment:
```suggestion
### What if my package seems like its category or domain is out of scope?
```

Reply:
Since this is `<h3>` it seems like it's mean to be part of the "domain" section (the last `<h2>`) but this also seems like it could apply to the "category" section (`<h2>` above this)